### PR TITLE
Create docker-image.yml

### PR DESCRIPTION
**Pull request by Its4Nik**
This allows automatic image building using a GitHub Workflow.

The built image will be available under the GitHub container registry (ghcr.io)

The image gets built when a new release is published, we could change the behavior to have Development images as well and/or always build an image when we push to the main branch.

### CODE REVIEW

**Comment by electronick** [comment]
@Its4Nik we really appreciate your effort on this one. It would be great if you could upgrade your PR a bit considering my comment above. Thanks!

**Comment by Its4Nik** [comment]
> @Its4Nik we really appreciate your effort on this one. It would be great if you could upgrade your PR a bit considering my comment above. Thanks!

I am unsure which comment you are referring to, I can't seem to find it...

**Review by electronick — CHANGES_REQUESTED** [review]
pmorelli92/github-container-registry-build-push should be updated to docker/build-push-action

File: .github/workflows/docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    contents: read
+
+jobs:
+    build-main:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+              name: Checkout repository
+
+            - uses: pmorelli92/github-container-registry-build-push@2.2.1
```
[the remark this answers is not in the document]
Comment (Its4Nik) [reply .github/workflows/docker-image.yml:18]: Im gonna adjust it accordingly

**Review by electronick — APPROVED** [review]
Thank you for your effort, @Its4Nik

**Pull request merged**